ProjectCoimbra.UWP/Project.Coimbra.Midi/DryWetMidiIntegration/MarkablePlayback.cs

```diff
@@ -397,6 +398,17 @@ private void OnClockTick(object sender, object eventArgs)
 
                 this.CheckDisplay(currentTime, playbackEvent);
                 this.CheckPlaySound(currentTime, playbackEvent);
+
+                var endTime = playbackEvent.Time.Add(new TimeSpan(playbackEvent.Event.DeltaTime));
+                if (largestEndTime < endTime)
```

Comment:
You could also use `largestEndTime = Math.Min(largestEndTime, endTime)`, but the cost of always resulting in an assignment. That said, perf probably isn't the biggest issue here so it should be fine.

Reply:
I wouldn't recommend because the variables are TimeSpan and we would need to create a new object from Ticks if we take the result of Min function.

Reply:
Good point, I forgot about that. I agree it makes sense to leave it as is given the data type.